pdns/dnsseckeeper.hh

```diff
@@ -162,8 +162,8 @@ public:
   keyset_t getEntryPoints(const DNSName& zname);
   keyset_t getKeys(const DNSName& zone, bool useCache = true);
   DNSSECPrivateKey getKeyById(const DNSName& zone, unsigned int id);
-  bool addKey(const DNSName& zname, bool setSEPBit, int algorithm, int bits=0, bool active=true);
-  bool addKey(const DNSName& zname, const DNSSECPrivateKey& dpk, bool active=true);
+  int addKey(const DNSName& zname, bool setSEPBit, int algorithm, int bits=0, bool active=true);
+  int addKey(const DNSName& zname, const DNSSECPrivateKey& dpk, bool active=true);
```

Comment:
`addKey()` returns a signed int, because it returns -1 in case of error, but the other `*Key()` functions take an unsigned int, and `LAST_INSERT_ID()` for example returns a `BIGINT UNSIGNED (64-bit)`. Perhaps addKey should at least return an int64_t, or a boolean to indicate success with an additional unsigned int& parameter to return the key?